kernel/lib/time.cpp

```diff
@@ -0,0 +1,90 @@
+/**
+ * @file time.hpp
+ * @author Michel (JMallone) Gomes (michels@utfpr.edu.br)
+ * @brief Time lib code 
+ * @version 0.1
+ * @date 2021-07-27
+ *
+ * @copyright Copyright the Panix Contributors (c) 2021
+ *
+ * References:
+ *              https://github.com/sidsingh78/EPOCH-to-time-date-converter/blob/master/epoch_conv.c
+ *              https://www.oryx-embedded.com/doc/date__time_8c_source.html
+ */
+#include <dev/rtc/rtc.hpp>
+#include <dev/tty/tty.hpp>
+#include <lib/stdio.hpp>
+#include <lib/time.hpp>
+namespace Time {
+TimeDescriptor::TimeDescriptor()
+{
+    toDate();
+}
+
+TimeDescriptor::TimeDescriptor(int seconds, int minutes, int hour, int day, int month, int year)
+    : _seconds(seconds)
+    , _minutes(minutes)
+    , _hour(hour)
+    , _day(day)
+    , _month(month)
+    , _year(year)
+{
+    // Initialize nothing.
+}
+TimeDescriptor::~TimeDescriptor()
+{
+    // Nothing to deconstruct
+}
+
+void TimeDescriptor::converterEpochToDate(uint64_t epochNumber)
+{
+    int a, b, c, day, month, year;
+    setSeconds(epochNumber % 60);
+    epochNumber /= 60;
+    setMinutes(epochNumber % 60);
+    epochNumber /= 60;
+    setHour(epochNumber % 24);
+    epochNumber /= 24;
+
+    a = ((4 * epochNumber + 102032) / 146097 + 15);
```

Comment:
Could you add comments to the right side of the code that explains what each step is doing?